cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -74,6 +74,8 @@ type Evaluate struct {
 	Runtime string `long:"runtime" description:"The runtime which will be used for the evaluation." default:"local" choice:"local" choice:"docker"`
 	// RuntimeImage determines the container image used for any container runtime.
 	RuntimeImage string `long:"runtime-image" description:"The container image to use for the evaluation." default:""`
+	// Parallel holds the number of parallel executed runs.
+	Parallel uint `long:"parallel" description:"Amount of parallel executed runs." default:"1"`
```

Comment:
State that this only affects docker.

Reply:
Though I would maybe make the logic different (maybe in a follow up). Cause now we have runs and parallel executions... Would be cool if `parallel` would be a boolean and it automatically distributes multiple runs across parallel executions?

Reply:
Yeah but runs and parallel are different.
For better result spread I want to have 5 runs of each model, while my server is able to run 15-20 parallel evaluations.

Making it exclusive for containerized runs and clarify it in the comment is the way to go I guess.